packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -31,28 +32,46 @@ export class SkipLink extends GOVUKFrontendComponent {
     this.$module = $module
 
     // Check for linked element
-    const $linkedElement = this.getLinkedElement()
-    if (!$linkedElement) {
-      return
+    try {
+      const $linkedElement = this.getLinkedElement()
+      this.$linkedElement = $linkedElement
+    } catch (cause) {
+      throw new MissingElementError(
+        'Skip link: the linked HTML element does not exist',
+        {
+          cause: cause instanceof Error ? cause : undefined
```

Comment:
For [JavaScript built-in: Error: cause](https://caniuse.com/mdn-javascript_builtins_error_cause) browser support are we happy treating this as optional?

If so, do you mind adding a bit of safety to the `GOVUKFrontendError` constructor?

i.e. Use feature detection, not a polyfill

```mjs
export class GOVUKFrontendError extends Error {
  name = 'GOVUKFrontendError'

  /**
   * @param {string} message 
   * @param {ErrorOptions} options
   */
  constructor(message, { cause, ...options }) {
    super(message, options)

    // Add cause property where supported
    if (cause && new Error('test', { cause }).cause === cause) {
      this.cause = cause
    }
  }
}
```

We'll need to:

1. ESLint ignore "ES2022 Error Cause is forbidden" for feature detection purposes
2. ESLint ignore "ES2018 rest/spread properties are forbidden" if Babel handles it already
3. Triple check my feature detection idea, maybe it needs a try/catch?

For 3) it's mainly because `'cause' in Error.prototype` didn't work 🙈

Reply:
Or we remove `{ cause }` for another day?

Reply:
I'm leaning toward leaving `{ cause }` for now - feels like p'raps we need a bit more thinking about how we make it work and can do that as part of iterating on these errors.

I've pushed a change to use the error messages to convey the error to the user, whaddya reckon?